NEWS.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 ## Changes
+- Added `group.by` parameter to `FindAllMarkers`, allowing users to regroup their data using a non-default identity class prior to performing differential expression ([#9550](https://github.com/satijalab/seurat/pull/9550))
+#' performing differential expression (see example); \code{"ident"} to use Idents
```

Comment:
looks like a copy-paste error here :smile:

Reply:
🤦

Reply:
Not a big deal, I'll need to update `NEWS.md` as part of the final release anyways.